src/garage/experiment/local_tf_runner.py

```diff
@@ -182,7 +193,7 @@ def obtain_samples(self, itr, batch_size):
             logger.log('Obtaining samples...')
         return self.sampler.obtain_samples(itr, batch_size)
 
-    def save(self, epoch, paths=None):
+    def _save(self, epoch, paths=None):
```

Comment:
hmm why is save a private API and restore a public API? is there a problem with save being public?

Reply:
`save()` is only used inside. What's the reason to set it to public?